emoji/src/main/java/com/vanniktech/emoji/listeners/OnEmojiLongClickedListener.java

```diff
@@ -0,0 +1,11 @@
+package com.vanniktech.emoji.listeners;
+
+import android.support.annotation.RestrictTo;
+import android.view.View;
+
+import com.vanniktech.emoji.emoji.Emoji;
+
+@RestrictTo(RestrictTo.Scope.LIBRARY)
+public interface OnEmojiLongClickedListener {
```

Comment:
make it package private and put it into the root if we do not expose this (to be consistent)